src/components/twind/withTwind.ts

```diff
@@ -5,6 +5,8 @@ import {LitElement} from 'lit';
 
 const colors = {
   'brand-light': 'var(--bc-color-brand, #196CE7)',
+  'brand-text-light': 'var(--bc-color-brand-text, #FFFFFF)',
+  'brand-text-dark': 'var(--bc-color-brand-text, #000000)',
```

Comment:
is it ok for light and dark to use the same variable?

Reply:
Maybe since its also possible to set --bc-color-brand-dark it should be possible to set --bc-color-brand-dark-text? I was only considering that you had one brand bg/text colour combination and wanted to override the text colour independent of dark mode. But there are cases where your brand might have different bg-color for light/dark mode and subsequently text.

Maybe it could be a chained optional along --bc-color-brand-text-dark, --bc-color-brand-text, #000000? same as for brand-dark